roles/cache_builder/defaults/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+offline_cache_dir: "/tmp/deepops"
+build_tar_file: true
+tar_file_path: "/tmp/deepops-archive.tar.bz2"
+deepops_repo_url: "https://github.com/NVIDIA/deepops"
+
+centos_iso_mirror: "http://mirrors.ocf.berkeley.edu/centos/7.6.1810/isos/x86_64"
+centos_iso: "CentOS-7-x86_64-Minimal-1810.iso"
+download_yum_repos: yes
+yum_repos_to_sync:
+- "libnvidia-container"
+- "nvidia-container-runtime"
+- "nvidia-docker"
```

Comment:
I think this should be `nvidia-docker2`

Reply:
The repo name is `nvidia-docker`, and is found at `https://nvidia.github.io/nvidia-docker`. It actually contains both the `nvidia-docker` and `nvidia-docker2` packages.

Reply:
Sorry, thought these were package names. Where do the repo URLs come from?

Reply:
All the URLs are in `roles/cache_builder/files/yum.conf`